src/components/cylc/gscan/GScan.vue

```diff
@@ -228,9 +230,7 @@ export default {
         GSCAN_DELTAS_SUBSCRIPTION,
         {},
         'root',
-        [
-          new GScanCallback()
-        ]
+        []
```

Comment:
Nice that these callbacks have gone. I was wondering about these when doing the log view.